Assets/_BForBoss/_Weapons/Scripts/ScriptableObjects/Weapons/Shotgun.asset

```diff
@@ -12,7 +12,10 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 681c7221818898543b23f8427a3a431f, type: 3}
   m_Name: Shotgun
   m_EditorClassIdentifier: 
-  _rateOfFire: 0.3
+  _rateOfFire: 0.8
   _bulletSpread: 0.5
-  _numberOfBullets: 10
-  _crosshairImage: {fileID: 21300000, guid: 5975143319e80f243aeae9a363a6e34f, type: 3}
+  _reloadDuration: 2
+  _bulletsPerShot: 1
```

Comment:
I think this should still be 10 bullets per shot, right?

Reply:
reverted it back to 10